### ISSUE TITLE
Support for Node 18/19

### ISSUE THREAD

**Issue by nickythorne** [teambit/bvm#80]
### Describe the bug
We are in the process of upgrading our tooling versions, including node to as latest version as possible. In our case this would be Node 19. We have updated our base docker images to derive from node:18-alpine. However, we are getting errors installing the latest version of bit cli within these images, relating to unsupported node version.

### Steps to Reproduce
Build Docker image from node:19-alpine3.15 and install bit

```
FROM node:18-alpine
RUN @teambit/bvm@latest && bvm install 
Build image docker build .
```

Get following error
`Node version 18.12.1 is not supported, please use Node.js >=14.19.3 <17.0.0.`

### Expected Behavior
Are there plans to support Node 18 (given that Node 19 is the current version)?

### Specifications
Bit version:
Workspace type: (harmony/legacy) harmony
Node version: 18.12.1
Platform: alpine linux

**Comment by jcfant-eb** [comment]
It's also pinned to a specific version, which isn't available on Brew.

Node.js version 16.16.0 is not installed. Try to reinstall the wanted bit CLI version with the --override option

**Comment by GiladShoham** [comment]
bit now supports node 18 (since version 0.2.3) (and shipped with node 18.16.1 by default)
